### DISCUSSION TITLE
Reduce Wifi range.

### DISCUSSION THREAD

**Discussion by Neogene** [cyberman54/ESP32-Paxcounter#1002] [General]
Hi, in the screen is displayed for wifi: pwr = 14. 

Which is the config var to reduce the power of the Wifi to reduce the area tracked? 

There is some range of min/max power to know (i'm using LilyGo 1.6.1) that can be used for this variable?

Thank you in advance.

**Comment by cyberman54** [comment]
1) did you read the documentation? Look for RSSI limit
2) if you see pwr on the display, you're using a very old outdated version of this project. You should update (reflash your board)
3) pwr display in older versions shows the LORAWAN output power level, is not related to Wifi in any way
4) To set Wifi reception rssi threshold, edit https://github.com/cyberman54/ESP32-Paxcounter/blob/master/shared/paxcounter_orig.conf#L33, then clear the NVRAM of an already flashed module (otherwise the stored value will be reloaded during boot), then build and flash the new code. RSSI level can also be set during runtime using an rcommand, transmitted via Lorawan, see documentation.

**Reply by Neogene** [reply]
Hi, the device reports on boot fw 3.6.1,  i run pio run -t it showed some log about erasing contents, then started with multiple dots and did timeout, i had to disconnect and reconnect to be able to see again the board. 

![IMG_8027](https://github.com/cyberman54/ESP32-Paxcounter/assets/2322707/17b6147a-e261-411b-8ca5-851f25937956)
![IMG_8028](https://github.com/cyberman54/ESP32-Paxcounter/assets/2322707/eba5e1e0-2171-43bf-9b83-8692a0e62771)
![IMG_8031](https://github.com/cyberman54/ESP32-Paxcounter/assets/2322707/c9d1a389-6498-49c5-9432-a7b52d803676)

**Reply by Neogene** [reply]
Ok i was using the build from the release area, i will download the current repo status to get the last version.

**Comment by cyberman54** [comment]
okay not i got it which pwr display you have seen. It's fully okay using the latest release. Again, pwr is showing lora output RF level.

**Reply by cyberman54** [reply]
RLIM shows current RSSI limit. It shows off what means there is no threshold for RSSI set.

**Reply by Neogene** [reply]
If I understand well changing the RSSILIMIT value is a way to reduce the number of the devices sniffed depending on the power of the signal received but if a device is far and has a high signal power i will detect it if my wifi is configured to use max power.
Did i get right? in that case can I configure the wifi to use less power to reduce its range/distance, and combine RSSLIMIT to track only devices with strong signal and close to my esp32?

**Comment by cyberman54** [comment]
RSSILimit controls the signal reception strength the sniffer is listening to. Not more, not less. You cannot conclude power or distance of the sender from this.

**Reply by Neogene** [reply]
No i mean there is a configuration to reduce the emissive power of the embedded wifi to make it's range smaller?

**Comment by cyberman54** [comment]
Did you understand the concept of wifi sniffing? The wifi is used in receiving mode only, it does not send, does not emit power.

**Reply by Neogene** [reply]
no, otherwise I wouldn't have asked these questions, I develop at a higher level and apart from some experiments with Arduino I have never had the opportunity to study the functioning of wifi and these concepts. So i’m just started to read the code this week.
